src/main/resources/application.properties

```diff
@@ -14,9 +14,15 @@ spring.datasource.username = ${DATABASE_USER}
 spring.datasource.password = ${DATABASE_PASSWORD}
 spring.jpa.database-platform=org.hibernate.dialect.Oracle12cDialect
 
+spring.flyway.locations = classpath:db/migration
```

Comment:
What do you think about having `migrations` instead of `migration`?

Reply:
I'd leave it as it is, since it's the [default value](https://flywaydb.org/documentation/configuration/parameters/locations#default). I think we can keep it unless we have a certain organization in min (for instance, [this](https://flywaydb.org/documentation/configuration/parameters/locations#wildcards)).

I wasn't even going to configure it. It's just there so that we know where to put the first scripts. Now that I think about it, I'm partial to *not* configuring it anymore.

Reply:
`db/migration` is the default value. Also, as the default value, it doesn't need to be passed.

Reply:
Sounds great!